platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/style/layer.junit.ejs

```diff
@@ -165,7 +165,12 @@ public class <%- camelize(type) %>LayerTest extends BaseActivityTest {
 <% if (property.tokens) { -%>
 
       layer.setProperties(<%- camelizeWithLeadingLowercase(property.name) %>("{token}"));
+<% if (property.type === 'formatted') { -%>
```

Comment:
I'd like to re-iterate concerns from https://github.com/mapbox/mapbox-gl-native/pull/12624#pullrequestreview-158773391 for Android as well. Will plain text, or other than `format` expressions accepted for `text-field` property? If yes, we need to rework this `.ejs` file to generate tests for both.

Reply:
Yeah plain text is accepted but may be transformed... 🤔

Reply:
Transformed to an expression? This would change the behavior of `PropertyValue#isExpression` when the `text-field` property is retrieved by `SymbolLayer#getTextField` which would return `false` even though it should return `true` for plain text. That's maybe a minor inconsistency but worth taking into consideration.